src/components/Layout/index.js

```diff
@@ -10,6 +10,14 @@ injectGlobal`
   html {
     font-family: 'Montserrat', sans-serif;
   }
+  @media screen and (max-width: 320px) {
```

Comment:
no need to present hyphens for regular layout as it seems useless, but long words on mobiles should be wrapped with it.